extensions/trace-utils/src/main/java/io/opentelemetry/extension/trace/MessageEvent.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.extension.trace;
-
-import static io.opentelemetry.api.common.AttributeKey.longKey;
-import static io.opentelemetry.api.common.AttributeKey.stringKey;
-
-import io.opentelemetry.api.common.AttributeKey;
-import io.opentelemetry.api.common.Attributes;
-import io.opentelemetry.api.common.AttributesBuilder;
-import io.opentelemetry.api.trace.Span;
-import javax.annotation.concurrent.Immutable;
-
-/**
- * A class that represents a generic messaging event. This class can represent messaging happened in
- * any layer, especially higher application layer. Thus, it can be used when recording events in
- * pipeline works, in-process bidirectional streams and batch processing.
- *
- * <p>It requires a {@link Type type} and a message id that serves to uniquely identify each
- * message. It can optionally have information about the message size.
- *
- * @deprecated This class will be removed in SDK 0.13.0. If these attributes are useful to you,
- *     consider filing an issue or a PR to have them added to {@link
- *     io.opentelemetry.api.trace.attributes.SemanticAttributes}.
- */
-@Immutable
-@Deprecated
-public final class MessageEvent {
```

Comment:
FYI: This is for backwards compatibility with OpenCensus, so cannot be removed, if we still want to achieve the goal of backwards compatibility :)

Reply:
This may leave in opencensus artifact, so ok to remove it.